### ISSUE TITLE
Add `List Alerts` for overview of alerts and their status

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#201910]
The command `List Audio Cues` allows for reviewing the available cues, configuring each, and checking the status (enabled/disabled). Now that we will have many configurable alerts, we should have a similar command for these. 

I will mention these commands in the help dialog, which I believe solves the issue of users not knowing how to disable them. 

cc @rperez030 and @jooyoungseo

**Comment by meganrogge** [comment]
One question is whether this info should appear in just the editor help dialog or all of them.

**Comment by rperez030** [comment]
Is it possible to disable audio cues directly from that list? i didn't know that was possible. How do I do that? They are not checkboxes, and do not respond to enter or the space bar.

**Comment by meganrogge** [comment]
You can tab to the settings gear buttons. Perhaps that's not discoverable enough?

**Comment by meganrogge** [comment]
@TylerLeonhardt have you had any issues related to not indicating that there are buttons to tab to for quick pick items?

**Comment by TylerLeonhardt** [comment]
@meganrogge no I haven't seen any issues related to that.

**Comment by rperez030** [comment]
It works, but in all my time using the editor, it never occurred to me to try using the tab key in that list. I always assumed it was read only.

**Comment by hediet** [comment]
This command and the audio cue command should be merged!

**Comment by meganrogge** [comment]
They will be in February @hediet 👍🏼
